common/test/acceptance/pages/lms/discussion.py

```diff
@@ -48,6 +48,7 @@ def _secondary_action_menu_open(self, ancestor_selector):
         Given the selector for an ancestor of a secondary menu, return a context
         manager that will open and close the menu
         """
+        self.wait_for_ajax()
```

Comment:
This is the main bug that was reported in TNL-4151

By not waiting for ajax callbacks to finish, the button we were trying to click in the following line was never populated, so nothing happened when we waited in the `EmptyPromise` after that.